api/src/services/email.service.ts

```diff
@@ -672,7 +672,8 @@ export class EmailService {
             appUrl: jurisdiction.publicUrl,
           },
           appUrl: jurisdiction.publicUrl,
-          // These two URLs are placeholders and must be updated per jurisdiction
+          // These three URLs are placeholders and must be updated per jurisdiction
```

Comment:
Maybe the comment should be updated now that we have a specific url

Reply:
I'm not following. Do you mean the notificationsUrl and helpCenterUrl will be identical?

Reply:
Oh sorry, I just meant that one of the three URLs with this change now has the non-placeholder URL so it's just the other two urls that need to be updated at a future date